alpine/Dockerfile

```diff
@@ -46,10 +48,10 @@ COPY --from=toolbox /toolbox $TOOLBOX_TARGET_DIR/toolbox
 COPY --from=composer:2 /usr/bin/composer $TOOLBOX_TARGET_DIR/composer
 COPY entrypoint.sh /entrypoint.sh
 
-RUN php $TOOLBOX_TARGET_DIR/toolbox install \
+RUN --mount=type=secret,id=composer.auth,target=${TOOLBOX_TARGET_DIR}/.composer/auth.json \
```

Comment:
Would it be better to use `${COMPOSER_HOME}/auth.json` ?

Reply:
I wasn't sure I could use an env there (rather than arg) but it works 👍